### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/FloatingToolbar.js [hunk 1]
```diff
@@ -79,8 +79,9 @@ export default function FloatingToolbar({buttons, item, itemRef}) {
 
 				{activeConfigurationPanel && (
 					<ConfigurationPanel
+						backgroundImageTitle={item.backgroundImageTitle}
 						configurationPanel={activeConfigurationPanel}
-						itemId={item.itemId}
+						itemConfig={item}
```
Comment (diegonvs) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/FloatingToolbar.js]: `itemConfig` is weird, your `item` makes more sense 😂

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/core/openImageSelector.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+	const itemSelectorDialog = new ItemSelectorDialog({
+		eventName: `${portletNamespace}selectImage`,
+		singleSelect: true,
+		title: Liferay.Language.get('select'),
+		url: imageSelectorURL
+	});
+
+	itemSelectorDialog.on('selectedItemChange', event => {
+		const selectedItem = event.selectedItem || {};
```
Comment (diegonvs) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/core/openImageSelector.js]: This usage of ItemSelectorDialog differs a bit from your current implementation but the `ItemSelector` component apparently is exactly that I was working on `openInfoItemSelector`.

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/FloatingToolbar.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/FloatingToolbar.js]: Yep! I think so 🙂

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ConfigurationPanel.js [hunk 3]
```diff
@@ -86,7 +95,10 @@ export default forwardRef(
 				{...otherProps}
 			>
 				<div className="p-3 popover popover-scrollable">
-					<ConfigurationPanelComponent itemId={itemId} />
+					<ConfigurationPanelComponent
+						backgroundImageTitle={backgroundImageTitle}
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ConfigurationPanel.js]: Passing backgroundImageTitle here is necessary? It looks a little bit too specific

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/core/openImageSelector.js (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/core/openImageSelector.js]: Right, it's almost the same 🙂 Maybe you can add a component similar to the one I implemented but for images, or even adapt the same component for working with both types 🤔

**Pull request closed without merging**